tests/avocado/avocado_qemu/test.py

```diff
@@ -379,6 +386,49 @@ def add_image(self, path, username=None, password=None, snapshot=True,
         if password is not None:
             self.password = password
 
+    def cloudinit(self, hostname=None, password=None):
+        """
+        Creates a CDROM Iso Image with the required cloudinit files
+        (meta-data and user-data) to make the initial Cloud Image
+        configuration, attaching the CDROM to the VM.
+
+        :param: hostname: The hostname to configure the image with
+        :param password: The password to configure the default user with
+        """
+        try:
+            geniso_bin = utils_path.find_command('genisoimage')
+        except:
+            raise QEMUCloudinitError('Command not found (genisoimage)')
```

Comment:
We should start documenting the binaries (and packages) that are hard-deps to avocado_qemu.

Reply:
indeed